pom.xml

```diff
@@ -45,8 +45,7 @@
     <connectorHost />
     <jenkins.host.address />
     <slaveAgentPort />
-    <!-- TODO fix KubernetesCloudTest todos once past 2.414 -->
-    <jenkins.version>2.401.1</jenkins.version>
+    <jenkins.version>2.415</jenkins.version>
```

Comment:
I imagine this bump is there to retrieve some necessary updates from core?

Cause it is higher than the recommended minimum version: https://www.jenkins.io/doc/developer/tutorial-improve/update-base-jenkins-version/

And also it does not align with the bom anymore (bom is using `2.401.x`).

If it is intended and mandatory to have `2.415` it could be interesting to write a comment explaining it and a TODO to align the bom as soon as available?

Reply:
Afair this is to get `<l:delete ...>`.

BOM should be bumped to the closest LTS which is `2.414.x`.

Reply:
The change was suggested by @Vlatombe, we need a version after https://github.com/jenkinsci/jenkins/pull/7658/, other than that ...